pkg/api/query/v1.go

```diff
@@ -494,6 +496,7 @@ func (qapi *QueryAPI) queryRange(r *http.Request) (interface{}, []error, *api.Ap
 		ResultType: res.Value.Type(),
 		Result:     res.Value,
 		Stats:      qs,
+		// TODO: inject storage Stats.
```

Comment:
I am in the process on adding this to API. I would definitely help on react side to render it nicely, maybe as hover box

Reply:
cc @onprem